code/datums/martial/boxing.dm

```diff
@@ -4,36 +4,55 @@
 	pacifist_style = TRUE
 
 /datum/martial_art/boxing/disarm_act(mob/living/attacker, mob/living/defender)
-	to_chat(attacker, span_warning("Can't disarm while boxing!"))
+	jab(attacker, defender)
 	return TRUE
 
 /datum/martial_art/boxing/grab_act(mob/living/attacker, mob/living/defender)
 	to_chat(attacker, span_warning("Can't grab while boxing!"))
 	return TRUE
 
 /datum/martial_art/boxing/harm_act(mob/living/attacker, mob/living/defender)
+	jab(attacker, defender)
+	return TRUE
 
+/datum/martial_art/boxing/proc/jab(mob/living/attacker, mob/living/defender)
 	var/mob/living/carbon/human/attacker_human = attacker
 	var/obj/item/bodypart/arm/active_arm = attacker_human.get_active_hand()
+	var/lower_force = active_arm.unarmed_damage_low
+	var/upper_force = active_arm.unarmed_damage_high
+	var/base_unarmed_effectiveness = active_arm.unarmed_effectiveness
+	var/attack_sound = active_arm.unarmed_attack_sound
+
+	var/obj/item/organ/internal/cyberimp/chest/spine/potential_spine = attacker_human.get_organ_slot(ORGAN_SLOT_SPINE)
+	if(potential_spine)
```

Comment:
this should be `istype(potential_spine)` in case different kinds of spine organs are added in the future (or someone varedits (or even luas, once I bring that back) an organ that goes in the spine slot)